x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -76,8 +76,18 @@ const ThreatDetailsViewComponent: React.FC<{
   enrichments: CtiEnrichment[];
   showInvestigationTimeEnrichments: boolean;
   loading: boolean;
+  /**
+   * Slot to render something before the beforeHeader
+   */
+  before?: React.ReactNode;
   children?: React.ReactNode;
-}> = ({ enrichments, showInvestigationTimeEnrichments, loading, children }) => {
+}> = ({
+  enrichments,
+  before = <EuiSpacer size="m" />,
```

Comment:
this feels weird to be to initialize this to the `EuiSpace`. I think I understand why you did it: you didn't want to modify the old code.
But int his case, as this component is used in a single place, what do you think about initializing to null, and have [this guy](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx#L361) add the `<EuiSpacer size="m" />`?
In our code we wouldn't have to pass `null`, which feels odd

Reply:
you got it:)